### ISSUE TITLE
Uploading png works, jpg fails

### ISSUE THREAD

**Issue by Pwelling** [helios-ag/FMElfinderBundle#275]
Hi there,

thank you for your plugin!

I'm experiencing strange behavior though on my production server, which I can not reproduce on my local machine.

When I upload a png file it succeeds (filesize 284kb, so tyni). However when I try to upload a jpg-file it fails.

I've set the max filesize and upload size to 8M in the php.ini, since the files can be up to 5 Mb. But this does not help

Both systems are running on Ubuntu (server 14.04, local 16.04, server is running with plesk with nginx in front as proxy and apache as backend.

Probably there is an isue with this system?

Please advise.

Oh.. site is built with Symfony3 and as said it works, except when I try to upload a .jpg file

In my config I have 
upload_allow: ['image/png', 'image/jpg', 'image/jpeg']
upload_deny: ['all']

I also tried with upload_allow on 'all', without success. 

In one of the logs on my server I also found:
2017/02/16 21:17:14 [crit] 717#0: *9022 open() "/var/lib/nginx/body/0000000258" failed (13: Permission denied), client: IP......, server: cms.irisopavontuur.nl, request: "POST /efconnect?mode= HTTP/1.1", host: "cms.irisopavontuur.nl", referrer: "http://cms.irisopavontuur.nl/elfinder?CKEditor=blog_blog&CKEditorFuncNum=1&langCode=nl"

As said, this does not effect the png-file

**Comment by helios-ag** [comment]
Hi @Pwelling, do you have any error in symfony logs?
But in my opinion it's purely nginx (or fpm) issue, but it's weird, as it allow only to upload certain type of files.

**Comment by Pwelling** [comment]
Thanks for your reply. I think the problem is in my server indeed, larger PNG-files are also blocked.

Probably has something to do with Plesk using Nginx as proxy and apache on the background. I haven't found the reason yet, but I will find it!

Thanks again

**Comment by Pwelling** [comment]
It took some time, but I finally found the reason.

As said, the server uses both Apache aswell as nginx, where nginx relais the sites from Apache.
It toook some time to find the correct solution since this one is very hard to tacle if you do not know what to look for. Hopefully it can help others who seem to ahve the same issue.

ok, now the cause and thus the solution.

Ownership of the nxinx-folder /var/lib/nginx was set to nginx:nginx.

Since the server itself is handled and owned by www-data, you get an acces-denied message
I changed ownership to the correct user and am now able to upload images

I found the final solution at https://github.com/dockerfile/nginx/issues/4
